src/Attribute/LevenshteinIndexLookup.php

```diff
@@ -167,7 +168,7 @@ public function getSuggestions($language, $pattern)
             $attributeIds
         );
         $query       = sprintf(
-            'SELECT DISTINCT word ' .
+            'SELECT DISTINCT word, tl_metamodel_levensthein_index.attribute ' .
```

Comment:
When prefixing one column, we could as well prefix the whole query to make it more readable again, however this is merely of cosmetic nature.

Reply:
prefixing and cahnge to queryBuilder comes in MM 2.2

Reply:
you just added a prefix in your fix :smiley_cat:

Reply:
maybe we have àttribute` in both tables...

Reply:
but not word?

Reply:
I`ll check prefixes in MM 2.2!